dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerErrorCode.java

```diff
@@ -71,6 +71,8 @@
     E1048( "Object: `{0}`, uid: `{1}`, has an invalid uid format." ),
     E1049( "Could not find OrganisationUnit: `{0}`, linked to Tracked Entity." ),
     E1050( "Event ScheduledAt date is missing." ),
+    E1053( "AttributeCategoryOptions `{0}` do not match the AttributeOptionCombo `{1}`." ),
```

Comment:
do not match or " ... is not part of " ?

Reply:
It could be both

if

`aoc.getCategoryOptions().containsAll( categoryOptions )
            && aoc.getCategoryOptions().size() == categoryOptions.size()`

is true, than we know that the AOC is the identifier of the given categoryOptions. If this condition is false, we know its not. The reason could be that 1. the COs are missing a CO or 2. that a CO is given that is not within the AOCs COs.

Does the wording then make sense? For the error to clearly represent the 2 error cases we would probably need another error code and more logic.